ansible/requirements.yml

```diff
@@ -123,8 +123,9 @@
 - src: openmicroscopy.sudoers
   version: 1.0.0
 
-- src: openmicroscopy.upgrade-distpackages
-  version: 1.0.1
+- name: openmicroscopy.upgrade-distpackages
+  src: https://github.com/manics/ansible-role-upgrade-distpackages/archive/excludes.tar.gz
```

Comment:
With the release of https://galaxy.ansible.com/openmicroscopy/upgrade-distpackages/, we should be able to drop this.